x-pack/plugins/cases/public/components/user_actions/property_actions/registered_attachments_property_actions.tsx

```diff
@@ -20,20 +20,22 @@ interface Props {
   isLoading: boolean;
   registeredAttachmentActions: AttachmentAction[];
   onDelete: () => void;
+  hideDefaultActions: boolean;
 }
 
 const RegisteredAttachmentsPropertyActionsComponent: React.FC<Props> = ({
   isLoading,
   registeredAttachmentActions,
   onDelete,
+  hideDefaultActions,
 }) => {
   const { permissions } = useCasesContext();
   const { showDeletionModal, onModalOpen, onConfirm, onCancel } = useDeletePropertyAction({
     onDelete,
   });
 
   const propertyActions = useMemo(() => {
-    const showTrashIcon = permissions.delete;
+    const showTrashIcon = permissions.delete && !hideDefaultActions;
```

Comment:
This is a good point!

We will want to enable the deletion action in this case(but not the default one since it is a file attachment and needs the modal).